src/suspectplacesearcher/main.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from geopy.distance import vincenty
+import requests
+import math
+
+
+class SuspectPlaceSearch(object):
```

Comment:
"suspect" does not tell much about the meaning we currently have. Is a place selling drugs? A front organization? I suggest something more direct such as "SexRelatedPlace".